ruby_event_store/lib/ruby_event_store/in_memory_repository.rb

```diff
@@ -113,8 +145,9 @@ def append(events, expected_version, stream_name, include_global)
           global_stream = read_stream_events_forward(GLOBAL_STREAM)
           raise EventDuplicatedInStream if global_stream.any? { |ev| ev.event_id.eql?(event.event_id) }
           global_stream.push(event)
+          @streams[GLOBAL_STREAM] = global_stream
```

Comment:
I don't like it because #270 and it makes it `all` stream equal to global stream which is not necessary in InMemory implementation.